### ISSUE TITLE
Not Working Full Screen and Translations for Quests and Dialogues.

### ISSUE THREAD

**Issue by Katherina13** [KC3Kai/KC3Kai#2039]
### Versions:
Chrome: Version 58.0.3029.96
OS: Windows 7 Ultimate x32
KC3 Kai: 31.6.2
Built: Chrome Web Store

### Actual behaviour:
The game is not at full screen and is not showing some translation from the dialogues as translation for the Quests.
This begin after the release 31.4.0
### Screenshots (if applicable):
![image](https://cloud.githubusercontent.com/assets/28518323/25786590/3feb6760-3355-11e7-914a-8b5a067a5416.png)
![image](https://cloud.githubusercontent.com/assets/28518323/25786562/d4a17e68-3354-11e7-95d9-3ae4ea07be25.png)

**Comment by mikenzb** [comment]
Have you made sure to enable `Apply Customizations` when you start up KC3?

**Comment by Katherina13** [comment]
You mean this one?
![image](https://cloud.githubusercontent.com/assets/28518323/25788960/fb53efd0-33e7-11e7-82a4-b1b3e4f06b23.png)

**Comment by mikenzb** [comment]
Yes, if that is enabled along with the other buttons `On-screen Subtitles` & `Quest Translations` and still doesn't work, make sure you don't have an adblocker enabled.
